zebROS_ws/src/ros_control_boilerplate/config/2023_compbot_base_jetson.yaml

```diff
@@ -0,0 +1,413 @@
+# ros_control_boilerplate Settings -----------------------
```

Comment:
Delete this file. We'll need it eventually, but not as part of this pull request